pkg/types/envd.go

```diff
@@ -31,6 +34,43 @@ const DefaultPathEnvUnix = "/opt/conda/envs/envd/bin:/opt/conda/bin:/usr/local/j
 // ';' character .
 const DefaultPathEnvWindows = system.DefaultPathEnvWindows
 
+const PythonBaseImage = "ubuntu:20.04"
+
+var EnvdSshdImage = fmt.Sprintf("tensorchord/envd-sshd-from-scratch:%s", version.GetEnvdVersion())
```

Comment:
Is there a good way to get the latest git tag?

Reply:
Is there any problem?

Reply:
`version.GetEnvdVersion()` should be okay for users. But for developers, this may return a dirty git tag.

Reply:
Fixed with `version.GetVersionForImageTag()`